on_startup.py

```diff
@@ -17,15 +18,44 @@
 
 
 class Tileset(object):
+    file_type = None
+    data_type = None
 
     def __init__(self, refinery_node):
         self.file_url = refinery_node[FILE_URL]
         self.file_name = refinery_node[FILE_URL].split("/")[-1]
         self.file_path = '{}{}'.format(DATA_DIRECTORY, self.file_name)
-        self.file_type = "cooler"
-        self.data_type = "matrix"
+        self._set_tileset_type_meta()
```

Comment:
look into pulling out download or possibly switching order
its not clear that _set_tileset_type_meta downloads things as well